p2p/server.go

```diff
@@ -1015,12 +1021,14 @@ func (srv *Server) launchPeer(c *conn) *Peer {
 		// to the peer.
 		p.events = &srv.peerFeed
 	}
+	srv.loopWG.Add(1)
```

Comment:
This is not a good change. The `loopWG` tracks the main loop goroutines of `p2p.Server`. The `Server.run` loop tracks the peers and waits for them to shut down. We cannot add the peer to `loopWG` because it introduces the possibility that `loopWG.Add` will be called concurrently with `loopWG.Wait`, which is not allowed.

Reply:
Is it OK to create a independent wg for all peers  when server close